lib/app/routes/app_router.dart

```diff
@@ -686,12 +686,12 @@ class AppRouter {
     GoRouterState routerState,
   ) async {
     if (state.status == AuthStatus.biometricCheck) {
-      context.pushNamed(
+      unawaited(context.pushNamed(
```

Comment:
I think this needs to be awaited because we want to wait for the push. 
But if we do it the same in other places you can keep it.